tools/mo/unit_tests/mo/frontend_ngraph_test.py

```diff
@@ -59,3 +59,12 @@ def test_main_test():
 
     status = subprocess.run(args, env=os.environ)
     assert not status.returncode
+
+
+def test_mo_fallback_test():
+    setup_env()
+    args = [sys.executable, '-m', 'pytest',
+            os.path.join(os.path.dirname(__file__), 'utils/mo_fallback_test.py'), '-s']
```

Comment:
should it be mo_fallback_test_actual.py? Otherwise CI may search for *_test.py pattern and will try to execute this file without proper 'setup_env'. But I see that now CI jobs already do appropriate environment setup before Model Optimizer UT, so if CI passes this can be good

Reply:
OK, I see. You are right that tests in the current config were run two times. Changed to _acutal.py convention.